### PR TITLE
Updated polymerapp for latest packages

### PR DESCRIPTION
**Pull request by kunaldeo**
Updated polymerapp template for polymer 0.15.5 and paper_elements 0.6.1. The template now uses key-up event because valueChanged was not working with the latest paper_elements package.

[Review on Reviewable](https://reviewable.io/reviews/google/stagehand/197)

### CODE REVIEW

File: templates/polymerapp/pubspec.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ environment:
   sdk: '>=1.0.0 <2.0.0'
 dependencies:
   browser: any
-  polymer: '>=0.15.0 <0.16.0'
-  paper_elements: '>=0.5.0 <0.6.0'
+  polymer: '>=0.15.5 <0.16.0'
```
Comment (devoncarew) [inline templates/polymerapp/pubspec.yaml]: Do we need these new lower limits or can we use `.0`?

File: templates/polymerapp/lib/main_app.dart [hunk 2]
```diff
@@ -2,18 +2,19 @@
 // is governed by a BSD-style license that can be found in the LICENSE file.
 
 import 'package:polymer/polymer.dart';
+import 'dart:html';
```
Comment (devoncarew) [inline templates/polymerapp/lib/main_app.dart]: nit: can you sort the `dart:` imports first?

**Comment by devoncarew** [comment]
Thanks for the PR! Comments in-line.

**Comment by kunaldeo** [comment]
Thanks for the comments. I have got the imports organized. Regarding version numbers, I have updated the paper elements to '>=0.6.1 <0.7.0' . 0.6.1 is the latest release of paper_elements as of now, thats why I have mentioned it. I will check between the releases if the sample breaks :)  .

**Comment by devoncarew** [comment]
It's generally good to keep version constraints wider when possible (unless we are using features or fixes from newer versions). This limits the possibility of creating an unsatisfiable set of version constraints (one dep needs polymer >=0.6.0 < 0.6.2, another polymer >= 0.6.4 < 0.7.0). If we do need polymer `>=0.15.5` and paper_elements >= 0.6.1, sgtm, I just want to make sure that that's the case. Thanks again for the PR!

**Comment by kunaldeo** [comment]
Got your point. After doing extensive testing (sorry for multiple commits) I have found out that , polymer version < 0.15.4 returns warning from ScriptCompactor "private symbols are not supported" and paper_elements  <0.6.1 breaks the sample altogether.

**Comment by devoncarew** [comment]
Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/google/stagehand/197)

Reviewed files:
- lib/generators/polymerapp_data.dart @ r3
- templates/polymerapp/lib/main_app.dart @ r2
- templates/polymerapp/lib/main_app.html @ r1
- templates/polymerapp/pubspec.yaml @ r3

---

Great! lgtm

---

**Comment by kunaldeo** [comment]
Thanks a lot for accepting the PR.

**Pull request merged**